source/reference/oplog-sizing.txt

```diff
@@ -40,55 +41,30 @@ Estimate ``oplog`` Size Needed for Initial Sync
       cluster. If there are multiple shards, the smallest number is the
       minimum ``oplog`` window.
 
-   .. step:: Estimate Copy Rate During Synching
-
-      .. _c2c-step-est-size:
-
-      To gather performance data while synching, start the sync process
-      and monitor how fast data is transferred between clusters.
-      
-      To start syncing, run the :ref:`/start <c2c-api-start>`
-      command.
+   .. step:: Determine mongosync replication lag
 
-      To get the ``copy_rate``:
-      
-      - run the  :ref:`/progress <c2c-api-progress>` command to get
-        ``estimatedCopiedBytes_time01``
-      - wait a second or two
-      - run the  :ref:`/progress <c2c-api-progress>` command to get
-        ``estimatedCopiedBytes_time02``
-      
-      The ``copy_rate`` is:
-
-      .. code-block:: shell
-
-         copy_rate = ( estimatedCopiedBytes_time02 - estimatedCopiedBytes_01) / time_between_requests
-
-   .. step:: Estimate Copy Time
-
-      Estimate the time needed to copy the entire collection. The
-      estimated copy time is:
-      
-      .. code-block:: shell
+      To get the ``lagTimeSeconds`` value, run the  
+      :ref:`/progress <c2c-api-progress>` command. 
+      The lag time is the time in seconds between the 
+      last event applied by ``mongosync`` and time of the current
+      latest event on the source cluster.
 
-         estimatedCopyTime = estimatedTotalBytes / copy_rate
+      It is a measure of how far behind the source cluster mongosync is.
 
    .. step:: Validate ``oplog`` Size
 
-      If the estimated time is larger than the minimum oplog window you
-      must cancel synchronization. Before restarting, make one of the
-      following changes:
+      If the lag time approaches the minimum ``oplog`` window, you
+      should make one of the following changes:
 
-      - Increase the oplog window. Use :dbcommand:`replSetResizeOplog`
-        to set ``minRetentionHours`` greater than the estimated copy
-        time.
-      - Scale up the ``mongosync`` instance. Add cpu or memory to scale
+      - Increase the ``oplog`` window. Use :dbcommand:`replSetResizeOplog`
+        to set ``minRetentionHours`` greater than the current ``oplog`` 
+        window.
+      - Scale up the ``mongosync`` instance. Add CPU or memory to scale
         up the ``mongosync`` node so that it has a higher copy rate.
 
 .. note::
 
-   The copy rate may vary during synchronization. To monitor progress,
-   repeat the :ref:`steps to estimate the copy rate
-   <c2c-step-est-size>` and verify that the copy rate stays about the
-   same.
+   The ``oplog`` window and rate of change of replication lag may vary during 
```

Comment:
(nit)
Maybe: "rate of replication lag change"
Or: "rate for change for replication lag"

Reply:
Fixed